File: Java/Baju.java
public class Baju extends Aksesoris {
    private String untuk;
    private String size;
    private String merk;

    public Baju(int id_produk, String nama_produk, int harga_produk, int stok_produk, String jenis, String bahan, String warna, String untuk, String size, String merk) {
        super(id_produk, nama_produk, harga_produk, stok_produk, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    @Override
    public String tampilkanInfo() {
        return super.tampilkanInfo() + "\t" + untuk + "\t" + size + "\t" + merk;
    }
}

File: PHP/Baju.php
<?php
require_once "Aksesoris.php";

class Baju extends Aksesoris {
    private $untuk;
    private $size;
    private $merk;

    public function __construct($id_produk, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna, $untuk, $size, $merk) {
        parent::__construct($id_produk, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    public function getInfo() {
        return array_merge(parent::getInfo(), [$this->untuk, $this->size, $this->merk]);
    }
}
?>
